styles/main.css

```diff
@@ -0,0 +1,651 @@
+.option:hover {
+  background-color: rgba(0, 0, 0, 0.03);
+}
+
+.trends .header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.trending-item {
+  display: flex;
+  justify-content: space-between;
+}
+/* MODALS */
+.modal {
+  background-color: var(--gray);
+}
+.more {
+}
+
+/* SUGGESTIONS */
+.suggestion-item {
+  display: grid;
+  grid-template-columns: min-content 1fr min-content;
+  align-items: center;
+}
+
+.sm-show {
+  display: none;
+}
+
+/* TWEETBOX */
+
+#text-container:focus-within {
+  border-bottom: 1px solid var(--border-color);
+}
+
+#text-container:focus-within .visibility {
+  display: flex;
+  align-items: center;
+}
+
+.visibility {
+  display: none;
+  margin-bottom: 10px;
+}
+
+textarea {
+  resize: none;
+  width: 100%;
+  vertical-align: center;
+}
+
+textarea:focus {
+  outline: none;
+}
+
+
+.icon-container {
+    width: 25px;
+    height: 25px;
+    border-radius: 50%;
+    overflow: hidden;
+    transition: all 0.1s ease-in-out;
+    box-sizing: content-box;
+    padding: 3px;
+    display: flex;
+    align-content: center;
+    align-items: center;
+    justify-content: center;
+
+}
+.tweet-option-inner {
+    display: flex;
+    align-items: center;
+}
+
+.tweet-option-inner span {
+    transition: all 0.1s ease-in-out;
+}
+.tweet-option-inner.blue .icon-container{
+    background-color: rgba(73, 154, 236, 0.0);
+}
+
+.tweet-option-inner.blue:hover  .icon-container{
+    background-color:rgba(73, 154, 236, 0.1);
+    fill: var(--blue)
+}
+.tweet-option-inner.blue:hover {
+
+    color: var(--blue);
+
+}
+
+.tweet-option-inner.green .icon-container{
+    background-color: rgba(62, 141, 89, 0);
+}
+
+.tweet-option-inner.green:hover  .icon-container{
+    background-color:rgba(62, 141, 89, 0.1);
+    fill: rgb(62, 141, 89)
+}
+.tweet-option-inner.green:hover {
+
+    color: rgb(62, 141, 89)
+
+}
+
+.tweet-option-inner.red .icon-container{
+    background-color: rgba(236, 73, 73, 0);
+}
+
+.tweet-option-inner.red:hover  .icon-container{
+    background-color:rgba(236, 73, 73, 0.1);
+    fill: rgb(236, 73, 73)
+}
+.tweet-option-inner.red:hover {
+
+    color: rgb(236, 73, 73)
+
+}
+
+
+
+
+
+@media screen and (max-width: 1300px) {
+  .sm-hide {
+    display: none;
+  }
+
+  .nav {
+    width: auto;
+  }
+  .container {
+    width: 950px;
+  }
+
+  .center-items {
+    justify-content: center;
+  }
+
+  .tweet-button {
+    box-sizing: border-box;
+    width: 50px;
+    height: 50px !important;
+    border-radius: 50%;
+  }
+
+  ul.nav {
+    text-align: center;
+  }
+
+  .sm-show {
+    display: block;
+  }
+}
+
+@media screen and (max-width: 950px) {
+  .aside-header {
+    display: none;
+  }
+
+  .aside-content {
+    display: none;
+  }
+
+  .content-header {
+    grid-template-columns: 600px;
+    width: 600px;
+  }
+
+  .content {
+    grid-template-columns: 600px;
+    width: 600px;
+  }
+
+  .container {
+    grid-template-columns: 1fr 600px;
+    width: min-content;
+  }
+}
```

Comment:
well structured, easy to read and understand

Reply:
I agree, I understand this code well because of the way it's structured.